builtin/providers/scaleway/provider.go

```diff
@@ -53,7 +53,7 @@ var scalewayMutexKV = mutexkv.NewMutexKV()
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	config := Config{
```

Comment:
please leave the `access_key` in for now, for backwards compatibility.

It could look like this:

```
apiKey := ""
if v, ok := d.Get("token").(string); ok {
  apiKey = v
}
if v, ok := d.Get("access_key").(string); ok {
  apiKey = v
}
…
Config{
  APIKey: apiKey,
}
```